clib/exninfo.ml

```diff
@@ -117,3 +117,10 @@ let capture e =
     e, add info backtrace_info bt
   else
     e, info e
+
+let reify () =
+  if !is_recording then
+    let bt = Printexc.get_raw_backtrace () in
```

Comment:
Wait a sec, I misread it the first time. I thought it was getting the call stack. Otherwise, this code doesn't make sense because `tclZERO` can be called outside of a try-with block.

Reply:
Indeed this should only happen when moving from `exn` to `tclZero`, I did a quick port of a quite old tree of mine and didn't think too much.

Reply:
Also reify is a horrible name for something that returns the current stack